### ISSUE TITLE
[BUG] onscroll's first tick is always "down"

### ISSUE THREAD

**Issue by druskus20** [elkowar/eww#278]
## Checklist before submitting an issue
- [x] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [x] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [x] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

## Describe the bug
When scrolling over a widget that has the "onscroll" property, the first tick is always "down", regardless of the direction of the scroll.

## Reproducing the issue
This is a simple widget to demonstrate the bug.
```
(defwidget scrollable []
    (eventbox :onscroll `notify-send {}`
        (label :text "scroll me!")))
```

## Expected behaviour
When scrolling over the widget, we will receive "down" or "up" notifications. Regardless of the direction we choose to scroll in, the first tick will always be "down".

**Comment by druskus20** [comment]
Actually, this is a gtk bug https://bugzilla.gnome.org/show_bug.cgi?id=675959

**Comment by elkowar** [comment]
Fixed by #280
